scala/providers.bzl

```diff
@@ -30,3 +30,10 @@ def create_scala_provider(
         transitive_runtime_jars = transitive_runtime_jars,
         transitive_exports = [] #needed by intellij plugin
     )
+
+JarsToLabels = provider(
```

Comment:
can we make a link to the bazel discussions about putting labels in the jars?

https://github.com/bazelbuild/bazel/issues/4584